src/factories/nodes.ts

```diff
@@ -13,18 +12,13 @@ import { WorkerLayoutMessage, WorkerMessage, layoutWorkerFactory } from '@/worke
 
 export type NodesContainer = Awaited<ReturnType<typeof nodesContainerFactory>>
 
-type Events = {
-  rendered: void,
-}
```

Comment:
I removed the factory specific events in favor of augmenting the built in pixi events. That way I can emit directly on containers which is simpler and avoids some event bubbling.